### PR TITLE
Add Koodo

### PR DESCRIPTION
**Pull request by jpayoung**
Add entry Koodo in _data/services.yml
- Icon (Koodo does not have a logo without text)
- No 2FA

### CODE REVIEW

File: _data/utilities.yml [hunk 1]
```diff
@@ -196,6 +196,12 @@ websites:
     doc: https://github.com/2factorauth/twofactorauth/pull/2304#issue-200730083
     exception: "2FA is only available for Swedish citizens with BankID."
 
+  - name: Koodo
+    url: https://www.koodomobile.com
+    twitter: Koodo
```
Comment (Carlgo11) [inline _data/utilities.yml]:
```suggestion
    twitter: koodo
```

File: _data/utilities.yml [hunk 2]
```diff
@@ -196,6 +196,12 @@ websites:
     doc: https://github.com/2factorauth/twofactorauth/pull/2304#issue-200730083
     exception: "2FA is only available for Swedish citizens with BankID."
 
+  - name: Koodo
```
Comment (Carlgo11) [inline _data/utilities.yml]:
```suggestion
  - name: Koodo Mobile
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
I suggest you use this image instead:

**Comment by Carlgo11** [comment]
@jpayoung you forgot to update the image :wink:

**Comment by jpayoung** [comment]
Ooops! Midterms have me tired 😛Updating now.

**Review by jpayoung — COMMENTED** [review]
@Carlgo11 Image updated! Let me know if there's anything else

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**